chainstate/src/detail/query.rs

```diff
@@ -177,52 +177,79 @@ impl<'a, S: BlockchainStorageRead, V: TransactionVerificationStrategy> Chainstat
         self.chainstate_ref.get_block_height_in_main_chain(id)
     }
 
-    pub fn get_headers(
+    fn get_headers_above(
```

Comment:
I really don't understand what "above" means... do you mean headers of descedants?